youtube_dl/extractor/facebook.py

```diff
@@ -655,6 +706,48 @@ def _extract_video_title(self, webpage, tahoe_data, video_id):
             video_title = 'Facebook video #%s' % video_id
         return video_title
 
+    def _extract_uploader_info_new_ui(self, uploader_json):
+        uploader_handle = self._search_regex(r'"name":"(.+?")', uploader_json, 'uploader')
+        uploader_url = self._search_regex(r'"url":"(.+?")', uploader_json, 'uploader_url')
+        uploader_url_str = uploader_url.decode("utf-8")
+        uploader = uploader_url_str.split('\\/')[-2]
+        return uploader_handle, uploader
+
+    def _extract_ids_info_new_ui(self, ids_json):
+        ids_json_str = ids_json.decode("utf-8")
+        ids = ids_json_str.split(':')
+        channel_id = ids[0]
+        video_id = ids[1]
+        return channel_id, video_id
+
+    def resolve_new_ui_live_info(self, webpage, tahoe_data):
+        live_status = 'not_live'
+        is_live_vod = self._search_regex(r'"isLiveVOD":(.+?),', tahoe_data.secondary, "vod_live")
+        if is_live_vod:
+            live_status ='live'
```

Comment:
live_status is one of 4: 
1. 'live' - the video is live now
2. 'completed' - the video is not live now but it was created as a live video
3. 'upcoming' - it is going to be a live video that will start soon
4. 'not_live' - none of the above ("regular" video)

Can we infer a more accurate live status in the new UI ? (take a look at how we infer the live status in the old UI)
For example, in the old UI we return is_live = live_status == 'live' looks like this is not the case here.
Please make sure you keep the old behavior as much as possible.